.github/README.md

```diff
@@ -0,0 +1,143 @@
+
+      steps:
+      - name: Get File Name Timestamp
+        id: filename_time
+        uses: nanzm/get-time-action@v1.1
+        with:
+          format: 'YYYYMMDD-HHmmss'
+
+      - name: Get File Content Timestamp
+        id: file_content_time
+        uses: nanzm/get-time-action@v1.1
+        with:
+          format: 'YYYY-MM-DDTHH:mm:ss.000000-05:00'
+
+      - name: Generate Filepath
+        id: fp
+        run: |
+          FILEPATH=.changes/unreleased/Dependencies-${{ steps.filename_time.outputs.time }}.yaml
+          echo "::set-output name=FILEPATH::$FILEPATH"
+  ```
+
+- Print out all input variables as the first step of a job.  This allows for easier debugging.
+
+  ```yaml
+  - name: Print variables
+    run: |
+        echo The last commit sha in the release: ${{ inputs.sha }}
+        echo The release version number: ${{ inputs.version_number }}
+        echo The changelog_path: ${{ inputs.changelog_path }}
+        echo The build_script_path: ${{ inputs.build_script_path }}
+        echo The s3_bucket_name: ${{ inputs.s3_bucket_name }}
+        echo The package_test_command: ${{ inputs.package_test_command }}
+  ```
+
+## Tips
+
+### Context
+- The GitHub cli is available in the default runners
+- Actions run in your context.  ie, using an action from the marketplace that uses the GITHUB_TOKEN uses the GITHUB_TOKEN generated by your workflow run.
+
+### Actions from the Marketplace
+- Don’t use external actions for things that can easily be accomplished manually.
+- Always read through what an external action does before using it!  Often an action in the GitHub Actions Marketplace can be replaced with a few lines in bash.  This is much more maintainable (and won’t change under us) and clear as to what’s actually happening.  It also prevents any
+- Pin actions _we don't control_ to tags. 
+
+### Connecting to AWS
+- Authenticate with the aws managed workflow
+
+  ```yaml
+  - name: Configure AWS credentials from Test account
+    uses: aws-actions/configure-aws-credentials@v1
+    with:
+      aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      aws-region: us-east-1
+  ```
+
+- Then access with the aws command that comes installed on the action runner machines
+
+  ```yaml
+  - name: Copy Artifacts from S3 via CLI
+    run: aws s3 cp ${{ env.s3_bucket }} . --recursive 
+  ```
+
+### Testing
+
+- Depending on what your action does, you may be able to use [act](https://github.com/nektos/act) to test the action locally.  Some features of GitHub actions do not work with act, among those are reusable workflows.  If you can't use act, you'll have to push your changes up before being able to test.  This can be slow.
+
+- View _all_ action output from the [actions](https://github.com/dbt-labs/dbt-core/actions) tab.  Workflow results last 1 year.
+
+- View the action output for your PR in the **Checks** tab of the PR.  This only shows the most recent run.
```

Comment:
Would it be useful to have consistent casing and bold formatting for the labs of the "Actions" and "Checks" tabs?
```suggestion
- View _all_ action output from the [**Actions**](https://github.com/dbt-labs/dbt-core/actions) tab.  Workflow results last 1 year.

- View the action output for your PR in the **Checks** tab of the PR.  This only shows the most recent run.
```